gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/openapi-apis.yaml

```diff
@@ -3221,8 +3221,8 @@ components:
                           example: 2023-05-25T12:40:46.184Z
                       validation:
                           $ref: "#/components/schemas/PlanValidation"
-            required:
-                - definitionVersion
+                  required:
```

Comment:
Are you sure about this ? DefinitionVersion comes from the GenericApi, not the `custom object`

Reply:
Forget about it, I just needed to open my 👀